neuralprophet/conformal.py

```diff
@@ -65,20 +66,20 @@ def predict(self, df: pd.DataFrame, df_cal: pd.DataFrame) -> pd.DataFrame:
                 df[f"yhat{step_number} - qhat{step_number}"] = df[f"yhat{step_number}"] - q_hat
                 df[f"yhat{step_number} + qhat{step_number}"] = df[f"yhat{step_number}"] + q_hat
             elif self.method == "cqr":
-                quantile_hi = str(max(self.quantiles) * 100)
                 quantile_lo = str(min(self.quantiles) * 100)
-                df[f"yhat{step_number} {quantile_hi}% - qhat{step_number}"] = (
-                    df[f"yhat{step_number} {quantile_hi}%"] - q_hat
-                )
-                df[f"yhat{step_number} {quantile_hi}% + qhat{step_number}"] = (
-                    df[f"yhat{step_number} {quantile_hi}%"] + q_hat
-                )
+                quantile_hi = str(max(self.quantiles) * 100)
```

Comment:
Because in the final forecast dataframe, it is easier to read from `quantile_lo` to `quantile_hi`. Therefore, I decided to swap the order so `quantile_lo` comes first before to `quantile_hi`.